pxr/usd/usdLux/lightAPI.h

```diff
@@ -289,9 +339,21 @@ class UsdLuxLightAPI : public UsdAPISchemaBase
     // --------------------------------------------------------------------- //
     // EXPOSURE 
     // --------------------------------------------------------------------- //
-    /// Scales the power of the light exponentially as a power
+    /// Scales the brightness of the light exponentially as a power
     /// of 2 (similar to an F-stop control over exposure).  The result
-    /// is multiplied against the intensity.
+    /// is multiplied against the intensity:
+    /// 
+    /// <center><b>
+    /// L<sub>Scalar</sub> = L<sub>Scalar</sub> ⋅ 2<sup>exposure</sup>
+    /// </b></center>
+    /// 
+    /// Normatively, the lights' emission is in units of spectral radiance
+    /// normalized such that a directly visible light with `intensity` 1 and
+    /// `exposure` 0 normally incident upon the sensor plane will generate a
+    /// pixel value of [1, 1, 1] in an RGB renderer, and thus have a luminance
+    /// of 1 nit (cd∕m²). A light with `intensity` 1 and `exposure` 2 would
+    /// therefore have a luminance of 4 nits.
```

Comment:
Ditto, can we have a think about radiometry here. Like to get rid of the standard observer concept.

Reply:
same answer, D series illuminant plus sensor exposure function, yes?